### PR TITLE
docs(fiatconnect): specification fixes

### PR DESCRIPTION
**Pull request by satish-ravi**
fixed typos, table of contents, numbering, links

### CODE REVIEW

File: fiatconnect-api.md [hunk 1]
```diff
@@ -2,245 +2,255 @@
 
 # 0. Table of Contents
 
+- [FiatConnect CICO Provider API Specification "RFC"](#fiatconnect-cico-provider-api-specification-rfc)
```
Comment (satish-ravi) [inline fiatconnect-api.md:5]: Generated this using a vscode md plugin which added this line too. Can remove if it is confusing.

File: fiatconnect-api.md [hunk 2]
```diff
@@ -2,245 +2,255 @@
...
+    - [8.1.3. `WebhookEventTypeEnum`](#813-webhookeventtypeenum)
+    - [8.1.4. `TransferStatusEnum`](#814-transferstatusenum)
+  - [8.2. Dynamic Definitions](#82-dynamic-definitions)
+    - [8.2.1. `FiatTypeEnum`](#821-fiattypeenum)
+    - [8.2.2. `CryptoTypeEnum`](#822-cryptotypeenum)
+    - [8.2.3. `KycSchemaEnum`](#823-kycschemaenum)
+    - [8.2.4. `FiatAccountTypeEnum`](#824-fiataccounttypeenum)
+    - [8.2.5. `FeeTypeEnum`](#825-feetypeenum)
+    - [8.2.6. `FeeFrequencyEnum`](#826-feefrequencyenum)
+    - [8.2.8. `FiatAccountSchemaEnum`](#828-fiataccountschemaenum)
```
Comment (cajubelt) [inline fiatconnect-api.md:229]: why 8 and not 7 here?

**Review by cajubelt — COMMENTED** [review]
looks good but wondering about the 8 vs 7 thing

File: fiatconnect-api.md (hunk 2 above)
Comment (satish-ravi) [reply fiatconnect-api.md]: fixed

**Review by jophish — APPROVED** [review]
Thanks so much for doing this! The updated TOC with correct links is a huge improvement.

**Pull request merged**